azure-pipelines.yml

```diff
@@ -73,6 +74,15 @@ jobs:
       Python39V3:
         pythonVersion: '3.9'
         workerPath: 'python/prodV3/worker.py'
+      Python37V4:
+        pythonVersion: '3.7'
+        workerPath: 'python/prodV4/worker.py'
```

Comment:
nit: shall we extract these paths into variables?

Reply:
Yes. Why didn't I think of this :)